Auth0/Auth0Authentication.swift

```diff
@@ -92,6 +92,11 @@ struct Auth0Authentication: Authentication {
             ]
         payload["username"] = username
         payload["user_metadata"] = userMetadata
+        if let rootAttributes = rootAttributes {
+            rootAttributes.forEach { (key, value) in
+                if payload[key] == nil { payload[key] = value }
```

Comment:
this behavior should be clarified in the method documentation

Reply:
That seems fair, it was intended to stop abuse like overwriting `client_id`

Reply:
That's no abuse IMHO. The user should be free to send whatever they want as "additional params". If they send something we already specify, that's their fault.

Reply:
I agree with you in principal.